client/app/components/api/sections/documents/DocumentView.jsx

```diff
@@ -0,0 +1,57 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+var RouteHandler = Router.RouteHandler;
+var Markdown = require('react-remarkable');
+
+var fetchDocument = function (data, callback) {
```

Comment:
This function should be part of the component or it should be moved to a helper file and required.